### DISCUSSION TITLE
Fail to login or register(validation not passing)

### DISCUSSION THREAD

**Discussion by alextanasa91** [codeigniter4/shield#652] [General]
hello guys.
I installed today the first time and after passing all the steps and i can not login or register

The Email Address field is required.
The Password field is required.

i look to the view input name ,to validation rules,i add new rules to app/config/validation and nothing works.

what should i do? what i make wrong?

**Comment by datamweb** [comment]
Hi, 
Are you a registered user?
 If yes, please run the following command 
```console
php spark env development
```
and then screenshot the login page?

**Reply by alextanasa91** [reply]
![Untitled2](https://user-images.githubusercontent.com/104349012/221000739-4054447b-a73d-47f8-ad82-ab8489244df9.png)
![Untitled](https://user-images.githubusercontent.com/104349012/221000744-2dbdc0de-b67d-4508-815a-6e31b931d6d0.png)
yes ,also ...if i want to register after submiting the form...the same error

**Reply by alextanasa91** [reply]
i installed the very first time in my life...also i look to youtube and make the same.but the error persist and the code looks fine with no changes

**Reply by datamweb** [reply]
This is not a common problem, can you tell me which version of shield you have installed?

Please explain step by step all the steps you have done from the beginning of installing the framework and shield so that I can check.

**Comment by datamweb** [comment]
Closed due to lack of response.
If there was a problem, please open it again.
